packages/core/src/browser/breadcrumbs/breadcrumb-popup-container-renderer.tsx

```diff
@@ -0,0 +1,122 @@
+/********************************************************************************
+ * Copyright (C) 2019 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+import { ReactRenderer } from '../widgets';
+import { injectable } from 'inversify';
+import { Breadcrumbs } from './breadcrumbs';
+import PerfectScrollbar from 'perfect-scrollbar';
+import { BreadcrumbPopup } from './breadcrumb-popup';
+
+export const BreadcrumbPopupContainerRenderer = Symbol('BreadcrumbPopupContainerRenderer');
+export interface BreadcrumbPopupContainerRenderer {
```

Comment:
Do we need interface, symbol and a class? Should not `BreadcrumbPopupContainerRenderer` be enough? Please see https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#classes-over-interfaces